app/lib/forms/choose_your_npq.rb

```diff
@@ -2,15 +2,33 @@ module Forms
   class ChooseYourNpq < Base
     include Helpers::Institution
 
-    attr_accessor :course_id
+    QUESTION_NAME = :course_identifier
```

Comment:
The course_id question goes away, replaced with course_identifier. Now users will select from the same options as previously except the value on each is a unique identifier that is consistent across databases and uncoupled from primary keys. You can see a list of courses and their identifiers in config/initializers/courses.rb.

These identifiers can then be used in l18n files reliably since they are always the same. The unreliability of primary keys is what lead to previous bugs in #676.